test/libyul/objectCompiler/long_object_name.yul

```diff
@@ -16,10 +16,14 @@ object "t" {
 //   0x00
 //     /* "source":23:147   */
 //   sstore
+//     /* "source":19:150   */
+//   stop
 // stop
 //
 // sub_0: assembly {
+//         /* "source":272:274   */
+//       stop
```

Comment:
Maybe we shouldn't always put a `stop` in the code?

Reply:
I think we should - it doesn't really hurt and it's required in most cases that aren't merely a single block of straight control flow anyways.